test/spec.js

```diff
@@ -0,0 +1,54 @@
+"use-strict";
+const Application = require('spectron').Application;
+const assert = require('assert');
+const electronPath = require('electron'); // Require Electron from the binaries included in node_modules.
+const path = require('path');
+
+const app = new Application({
+	path: electronPath,
+	args: [path.join(__dirname, "..", "dist/js/main", "main.js"), "--dev", "--no-sandbox"],
+});
+
+describe('Application launch', function() {
+	this.timeout(10000);
+
+	beforeEach(() => {
+		return app.start();
+	});
+
+	afterEach(() => {
+		if (app && app.isRunning()) {
+			return app.stop();
+		}
+	});
+
+	it("shows an initial window", () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['it' is not defined.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5176720)